styles/components/_modal.scss

```diff
@@ -12,12 +12,12 @@
   align-items: center;
   justify-content: center;
   // Color /
-  background-color: rgba(0, 0, 0, 0.8);
+  background-color: rgba(0, 0, 0, 0.5);
   &__container {
     // Display & Box Model /
     display: flex;
-    width: 80%;
-    height: 80%;
+    // width: 80%;
+    // height: 80%;
```

Comment:
So the modal will always be depending on the content, not on the size of the screen?

Reply:
I think this will look better. I.e. on win modal there would be large blank space around if it would depend on screen size. And if we make it smaller, then trade modal can be squeezed.

Reply:
Well, I guess it would be important if we were considering becoming more "RWD", cause then breakpoints would be important. I thought it would be better for all modals to be the same, but you have a point, the win modal has very little content in comparison to the others.